src/Go/effects/netscriptGoImplementation.ts

```diff
@@ -389,7 +386,7 @@ export function cheatSuccessChance(cheatCount: number) {
 /**
  * Attempts to remove an existing router from the board. Can fail. If failed, can immediately end the game
  */
-export function cheatRemoveRouter(
+export async function cheatRemoveRouter(
```

Comment:
None of these cheat functions should be async. Also, they're all missing a return type.